java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -523,12 +524,45 @@ private static Object[] scriptingAndFunctionsCommands(BaseTransaction<?> transac
         final String code =
                 "#!lua name=mylib1T \n"
                         + " redis.register_function('myfunc1T', function(keys, args) return args[1] end)";
+        var expectedFuncData =
+                new HashMap<String, Object>() {
+                    {
+                        put("name", "myfunc1T");
+                        put("description", null);
+                        put("flags", Set.of());
+                    }
+                };
 
-        transaction.functionLoad(code).functionLoadReplace(code);
+        transaction
+                .customCommand(new String[] {"function", "flush", "sync"})
+                .functionList()
+                .functionListWithCode()
+                .functionLoad(code)
+                .functionLoadReplace(code)
+                .functionList("otherLib")
+                .functionListWithCode("mylib1T")
+                .customCommand(new String[] {"function", "flush", "sync"});
 
         return new Object[] {
+            OK, // customCommand("function", "flush", "sync")
+            new Map[0], // functionList()
+            new Map[0], // functionListWithCode()
             "mylib1T", // functionLoad(code)
-            "mylib1T" // functionLoadReplace(code)
+            "mylib1T", // functionLoadReplace(code)
+            new Map[0], // functionList("otherLib")
+            new Map[] {
```

Comment:
can you save this Map[] to a variable instead? 
```suggestion
            myLib1TMap, // functionListWithCode("mylib1T")
```